volttron/platform/dbutils/mysqlfuncts.py

```diff
@@ -300,6 +305,103 @@ def query(self, topic_ids, id_name_map, start=None, end=None, skip=0,
                 cursor.close()
         return values
 
+    @contextlib.contextmanager
+    def bulk_insert_data(self):
+        """
+        This function implements the bulk insert requirements for Redshift historian by overriding the
```

Comment:
This comment is incorrect as this is the mysql functs class.

Reply:
fixed on all functs classes